models/stdp_triplet_connection.h

```diff
@@ -185,7 +187,7 @@ class STDPTripletConnection : public Connection< targetidentifierT >
   facilitate_( double_t w, double_t kplus, double_t ky )
   {
     double_t new_w = w + kplus * ( Aplus_ + Aplus_triplet_ * ky );
-    return new_w < Wmax_ ? new_w : Wmax_;
+    return new_w < std::abs( Wmax_ ) ? new_w : std::abs( Wmax_ );
```

Comment:
Won't implementation along these lines permit sign switching? As in an excitatory connection reducing, going negative and turning into an inhibitory one and vice versa? 

Example - start with 5,500 - depress and turn into -5, 500 -> check still passes?
Example - start with -5, -500 - facilitate and turn into 5,-500 -> check still passes?